liquibase-standard/src/main/java/liquibase/sqlgenerator/core/CreateTableGenerator.java

```diff
@@ -403,6 +403,11 @@ public Sql[] generateSql(CreateTableStatement statement, Database database, SqlG
         if ((database instanceof MySQLDatabase) && (statement.getRemarks() != null)) {
             sql += " COMMENT='" + database.escapeStringForDatabase(statement.getRemarks()) + "' ";
         }
+
+        if (statement.isRowDependencies() && database.supportsCreateRowDependencies()) {
```

Comment:
Replace by instance of as in line 403 would be fine .

Reply:
Done.